server/routes/auth.routes.js

```diff
@@ -119,14 +134,15 @@ function getGovernmentEmail(emails) {
 }
 
 function createUser(loginGovUser) {
-  let now = new Date()
-  let date = (now.getMonth() + 1) + "-" + now.getDate() + "-" + now.getFullYear()
-  
-  //console.log("Login.gov user:", loginGovUser)
-
-  const gov_email = getGovernmentEmail(loginGovUser.all_emails || [])
+  logger.log('info', 'Creating new Login.gov user', {
+    email: loginGovUser.email,
+    office: getOfficeFromEmail(loginGovUser.email),
+    tag: 'createUser'
+  })
 
-  const user_email = gov_email || loginGovUser.email
```

Comment:
Why did you remove the user_email? It gets placed in the user_data.

Reply:
You're right - I mistakenly removed something that's used later... I added back the user_email variable before creating the user_data object